### ISSUE TITLE
Can not download !

### ISSUE THREAD

**Issue by zzndb** [emuirom/emui#1]
Can not download G520-T10's kernel, and any other device's.

**Comment by DavidWithTuxedo** [comment]
> 

Indeed, because this is only a collection of download links from "EMUI.COM", but that website has been closed.

**Comment by DavidWithTuxedo** [comment]
> Can not download G520-T10's kernel, and any other device's.

Hi,Bro! Do you have kernel source of C8816? I'm now here in search of it.

In regard to the "G520-T10", you can find its kernel source here https://github.com/YuxuanZuo/android_kernel_huawei_mt6589/releases/tag/v3.4.5
